### PR TITLE
Promote database backup logging to Info level

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

**Review by PiotrTrzpil — APPROVED** [review]

File: core/services/periodicbackup/backup.go [hunk 1]
```diff
@@ -71,13 +72,13 @@ func NewDatabaseBackup(config Config, logger logger.Logger) DatabaseBackup {
 	if config.DatabaseBackupDir() != "" {
 		dir, err := filepath.Abs(config.DatabaseBackupDir())
 		if err != nil {
-			logger.Errorf("Invalid path for DATABASE_BACKUP_DIR (%s) - please set it to a valid directory path", config.DatabaseBackupDir())
+			lggr.Errorf("Invalid path for DATABASE_BACKUP_DIR (%s) - please set it to a valid directory path", config.DatabaseBackupDir())
```
Comment (jmank88) [inline core/services/periodicbackup/backup.go:75]: This isn't a failure condition?

Comment (samsondav) [reply core/services/periodicbackup/backup.go:75]: Yes, this turned out to actually be more in depth than I thought. Working on a proper fix.

**Review by pinebit — APPROVED** [review]

File: core/services/periodicbackup/backup.go (hunk 1 above)
Comment (jmank88) [reply core/services/periodicbackup/backup.go:75]: Still no explicit return/error here? Will it error out on the empty `outputParentDir` later or something?

**Review by jmank88 — APPROVED** [review]

File: core/services/periodicbackup/backup.go (hunk 1 above)
Comment (samsondav) [reply core/services/periodicbackup/backup.go]: Good catch, this should only fail if `os.Getwd()` fails which imo warrants a `Fatal`

**Review by jmank88 — APPROVED** [review]

**Pull request merged**